client/homescreen/activity-panel/index.js

```diff
@@ -42,32 +48,48 @@ export const ActivityPanel = () => {
 	const panels = getAllPanels( panelsData );
 
 	return (
-		<Accordion>
-			<Fragment>
-				{ panels.map( ( panelData ) => {
-					const {
-						className,
-						count,
-						id,
-						initialOpen,
-						panel,
-						title,
-						collapsible,
-					} = panelData;
-					return (
-						<AccordionPanel
-							key={ id }
-							className={ className }
-							count={ count }
-							initialOpen={ initialOpen }
-							title={ title }
-							collapsible={ collapsible }
-						>
-							{ panel }
-						</AccordionPanel>
-					);
-				} ) }
-			</Fragment>
-		</Accordion>
+		<Panel className="woocommerce-activity-panel">
+			{ panels.map( ( panelData ) => {
+				const {
+					className,
+					count,
+					id,
+					initialOpen,
+					panel,
+					title,
+					collapsible,
+				} = panelData;
+				return collapsible ? (
+					<PanelBody
+						title={ [
+							<Text key={ title } variant="title.small">
+								{ title }
+							</Text>,
+							count !== null && <Badge count={ count } />,
+						] }
+						key={ id }
+						className={ className }
+						initialOpen={ initialOpen }
+						collapsible={ collapsible }
+						disabled={ ! collapsible }
+					>
+						<PanelRow>{ panel }</PanelRow>
+					</PanelBody>
+				) : (
+					<div className="components-panel__body">
```

Comment:
Hmm, so this is also something that might break based on version of `@wordpress/components` because we're relying on trying to imitate the existing markup of the panel component. Have you considered manually styling this component? If you provide the necessary styles in our library then it will increase size of styles a bit, but it should not break if the Panel component changes between versions.

Reply:
100% agreed.  Unfortunately, `@wordpress/components` does not expose a way for us to pass props to the button used here (`PanelBodyTitle` is not exported).  Just overriding via CSS I believe creates an accessibility issue.

This is brittle, but I think a change needs to happen upstream to remove this.

Reply:
Sounds like a fair trade off to me.